### PR TITLE
Ported integration test to osl_mysql_test resource

### PR DESCRIPTION
**Pull request by zwall-bp**
***Warning***: It seems that the `Dovecot` cookbook we are using is relying on the deprecated `ohai` cookbook. I've had to suppress deprecations as errors in order for this to converge properly.

I see a previous user was attempting to fix this issue in PR #22.

### CODE REVIEW

File: kitchen.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ provisioner:
   # multiple_converge: 2
 
   # ohai_plugin from the dovecot cookbook should declare `unified_mode true`
-  deprecations_as_errors: true
+  # deprecations_as_errors: true
```
Comment (ramereth) [inline kitchen.yml]: Just change this to false and add a comment as to why

**Review by ramereth — CHANGES_REQUESTED** [review]

**Review by ramereth — APPROVED** [review]
Patch bump

**Comment by zwall-bp** [comment]
!bump patch ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v2.0.3.  Have a nice day!

**Comment by zwall-bp** [comment]
Accidental push, ignore.

**Pull request merged**